src/appshell/view/internal/splashscreen/newinstanceloadingscreenview.cpp

```diff
@@ -30,17 +30,20 @@
 
 using namespace mu::appshell;
 
-NewInstanceLoadingScreenView::NewInstanceLoadingScreenView(const QString& openingFileName, QWidget* parent)
+NewInstanceLoadingScreenView::NewInstanceLoadingScreenView(bool forNewScore, const QString& openingFileName, QWidget* parent)
     : QWidget(parent)
 {
     setAttribute(Qt::WA_TranslucentBackground);
 
-    if (openingFileName.isEmpty()) {
+    if (forNewScore) {
```

Comment:
I don't quite understand the purpose of this flag. Could you please explain? Why can't we use the old condition?

Reply:
When launching a new instance, there are three options:
1. It's because the user creates a new score -> `forNewScore` is true
2. It's because an existing score is being opened, and the name is known before the score is opened -> `!openingFileName.isEmpty()` is true
3. It's because an existing score is being opened, but the name if not known before it is opened; this is the case when opening a musescore:// URL on Windows: Windows will launch MuseScore4.exe with the URL as a command line argument, so the score name is not known. Then we'll fall back to a general message like `Loading score…`.

Reply:
Please add a comment here with this information

Reply:
Can we implement calling MuseScore with a URL that contains parameters? This way, the website could pass both the ID and the title to us.

like musescore://open-score?id=12345678&title="hello"

Reply:
@Eism That would be possible, but I don't think that's really worth the effort (especially since I still hope that the whole multi-instances thing is temporary anyway)